functions/visualizations.py

```diff
@@ -17,6 +17,8 @@
     get_blob_as_stream,
 )
 
+CLUSTERGRAMMER_MAX_CATEGORY_CARDINALITY = 10
```

Comment:
It might be worth documenting the purpose of this with a comment. As I understand it, this is a rule of thumb that helps us automatically select categorical-ish clinical variables and exclude probably quantitative ones?